src/App.js

```diff
@@ -1,16 +1,64 @@
-import React from 'react';
-import styles from './App.module.css'
-import logo from './logo.svg';
-import Form from './Form.js'
+import React, { Component } from 'react'
+import './App.css'
+import Todos from './components/Todos'
+import Header from './components/Header'
+import AddTodo from './components/AddTodo'
 
+export default class App extends Component {
+  state = {
+    todos: [
+      {
+      id: 1,
+      title: 'Untold story',
+      completed: false
+      },
 
-function App() {
-  return (
-    <div className={styles.base}>
-      <Form />
+      {
+        id: 2,
+        title: 'Take out the trash',
+        completed: false
+      },
+        
+      {
+        id: 3,
+        title: 'Real life',
+        completed: false
+      },
+    ]
+  }
 
-    </div>
-  );
+
+  markComplete = (id) => {
+    console.log(id)
+  }
+
+  delTodo = (id) => {
+    this.setState({todos:[...this.state.todos.filter(todo => todo.id !== id)] });
+
+  }
+
+  addTodo = (title) => {
+    const newTodo = {
+      id: 4,
```

Comment:
use this  var next=this.state.todos[this.state.todos.length-1].id to take id in order